packages/editor/src/components/provider/index.js

```diff
@@ -154,8 +158,14 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 			type,
 			id
 		);
+
```

Comment:
I was playing with passing the pattern id via  the query param as the hope that the browser back button would work with this pushed to the history, but it doesn't for some reason, so we are probably better off passing this via the store somehow 🤔